app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+
+    @comment = @post.comments.new(text: comment_params[:text], user_id: current_user.id, post_id: @post.id)
```

Comment:
- [ ] You should be using `comment_params` to make sure that only allowed params are assigned. 
```ruby
def create
    @comment = @current_user.comments.new(comment_params)
    @comment.post_id = params[:post_id]
...
```